module-4/backend/week_two.md

```diff
@@ -4,9 +4,35 @@ Fork this respository. Answer the questions to the best of your ability. Try to
 
 Note: When you're done, submit a PR. 
 
-1. What is Webpack and why is it useful?
-2. When do you want to use event delegation?
+1. What is Webpack and why is it useful? 
+  - A frontend frame work to allow a platform for a server, and webtessting. Its useful because you can then update the application with live changes 
+2. When do you want to use event delegation? 
+  - Any time you "go" out to do something that takes more time. Like an api reqpuest
```

Comment:
event delegation is when we put event listeners on parent elements and then because of event bubbling, the event listener will impact the children specified. it's especially useful when you want to bind an event to an element that's loaded dynamically